src/finch/algebra/algebra.py

```diff
@@ -636,3 +635,23 @@ def sum_init_value(t):
 
 register_property(min, "__call__", "init_value", lambda op, arg: type_max(arg))
 register_property(max, "__call__", "init_value", lambda op, arg: type_min(arg))
+
+for trig_op in (
+    math.sin,
+    math.cos,
+    math.tan,
+    math.sinh,
+    math.cosh,
+    math.tanh,
```

Comment:
I think we can use solely numpy, for consistency.

Reply:
That said, I think it's okay to define properties for math functions we don't use, someone may want them later.